testing/test.js

```diff
@@ -0,0 +1,10 @@
+const puppeteer = require('puppeteer');
+const pageUrl = 'http://localhost:3418/';
+
+describe('ensure testing config', () => {
+
+  test('jest config', () => {
```

Comment:
Awesome test! jkjk but It's great that you have a testing framework in place that you can later add to

Reply:
I would love to chat about testing one of these days. I'm trying to get better about writing tests, but it feels a little daunting.